### DISCUSSION TITLE
Replace email with username in authentication

### DISCUSSION THREAD

**Discussion by lucacalcaterra** [filamentphp/filament#1692] [Help] [answered]
Hi,

i am trying to manage my new app wih filament and ldaprecord ... now i need to replace email with username field in authentication..

How can i achieve as simply as possible it  without rewrite a livewire page for auth ?

Thanks !
Luca

**Comment by danharrin** [comment, marked as answer]
Filament provides you with a simple authentication setup for your admin panel already, it's not been built to cover every case you may have. We've made it really easy to replace the default login page with your own in `config/filament.php`, so please take advantage of that.

**Reply by lucacalcaterra** [reply]
Thank @danharrin , i replaced with laravel/ui auth scaffolded pages. 👍

**Comment by yaplej** [comment]
@lucacalcaterra I am trying the same thing.  Were you ever able to get Filament's login page working with the LdapRecord's Auth Provider Model `LdapRecord\Models\ActiveDirectory\User::class`?

[So far](https://github.com/laravel-filament/filament/discussions/2493) I have only been able to redirect back to the laravel/ui auth page.
